packages/grafana-ui/src/components/Table/TableNG/TableNG.tsx

```diff
@@ -208,8 +208,9 @@ export function TableNG(props: TableNGProps) {
             justifyContent={justifyColumnContent}
           />
         ),
-        width: columnWidth,
-        minWidth: columnMinWidth,
+        // TODO these anys are making me sad
+        width: field.config.custom.width ?? columnWidth,
+        minWidth: field.config.custom.minWidth ?? columnMinWidth,
```

Comment:
If we get option directly from field maybe to simplify a bit:

```suggestion
        width: field.config.custom.width ?? 'auto',
        minWidth: field.config.custom.minWidth ?? COLUMN_MIN_WIDTH,
```

And replace `columnWidth` and `columnMinWidth` with:
```javascript
    useEffect(() => {
        setRevId(revId + 1);
    }, [fieldConfig]); // eslint-disable-line react-hooks/exhaustive-deps
```

@drew08t Let me know what do you think.